client/src/web/WebRTC.ts

```diff
@@ -101,6 +101,7 @@ export default class WebRTC {
   // method to add new video stream to videoGrid div
   addVideoStream(video: HTMLVideoElement, stream: MediaStream) {
     video.srcObject = stream
+    video.playsInline = true
```

Comment:
Nice catch! I've been wondering why it doesn't work on some devices.

Reply:
https://developer.apple.com/documentation/webkit/delivering_video_content_for_safari#3030251